openapi_python_client/schema/openapi_schema_pydantic/schema.py

```diff
@@ -34,8 +34,10 @@ class Schema(BaseModel):
     uniqueItems: Optional[bool] = None
     maxProperties: Optional[int] = Field(default=None, ge=0)
     minProperties: Optional[int] = Field(default=None, ge=0)
-    required: Optional[List[str]] = Field(default=None, min_items=1)
-    enum: Union[None, List[Optional[StrictInt]], List[Optional[StrictStr]]] = Field(default=None, min_items=1)
+    required: Optional[List[str]] = Field(default=None, min_length=1)
+    enum: Union[None, List[Optional[StrictInt]], List[Optional[StrictStr]], List[Optional[bool]]] = Field(
+        default=None, min_length=1
+    )
```

Comment:
This technically closes #785 but creates a bug—it will attempt to deserialize bools via their string value, namely `"True"` and `"False"`, rather than their JSON values `true` and `false`. So we need to roll this back for now until we can implement the rest of the logic for bool enums.